app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -140,7 +144,25 @@ class DrawActivity : AppCompatActivity() {
             } else if (!it.isLayerSheetOpen) {
                 isLayerListViewSheetOpen = false
             }
+            if (it.isColorPickerOpen && !isColorPickerSheetOpen) {
```

Comment:
What is the different of the picker and the sheet?

Reply:
Renamed them both to reflect that it is a sheet. There was no difference.